installation/ilmt-track-bamlcns-kubernetes.adoc

```diff
@@ -0,0 +1,37 @@
+= Tracking BAMOE license use in Kubernetes
+include::../partials/attributes.adoc[]
+
+
+== Prerequisites
+Install the License Service by following the https://www.ibm.com/docs/en/cloud-paks/foundational-services/3.23?topic=software-installing-license-service-stand-alone[tutorial].
+
+== Tracking BAMOE License
+After installing BAMOE Canvas and its dependencies following the installation tutorial, run the following commands depending on where BAMOE Canvas was installed.
+
+* OpenShift
+----
```

Comment:
[source, shell]

That should be added above line 12 to provide the proper highlighting.